src/test/java/org/cactoos/scalar/PropertiesOfTest.java

```diff
@@ -107,24 +102,4 @@ void convertsMapToProperties() {
         ).affirm();
     }
 
-    @Test
-    void sensesChangesInMap() throws Exception {
```

Comment:
@andreoss here too

Reply:
@victornoel This test (and the one above) checks either map is lazily created or not, thus it's removed

Reply:
@andreoss ok, anyway, those tests are so strange that even if they had sense they would be unmaintainable ^^ thx :)